### DISCUSSION TITLE
Lose 500 commits on my GitHub profile

### DISCUSSION THREAD

**Discussion by 0xMRTT** [community/community#29247] [Other Feature Feedback, Questions, & Ideas]
Yesterday i had more than 2100 commits. Today, while changing my status I saw that I have only 1600 commits.  Between today and yesterday I didn't delete anything... Where are my 500 commits.?

Lowlighter/metrics still have an highest number of commits (1800). It always been under the GitHub commit number.

Thanks in advance for any responses.

**Comment by airtower-luna** [comment]
Did you remove an email address from your account? Author email is how GitHub detects which commits are yours.

**Reply by 0xMRTT** [reply]
No

**Reply by igavelyuk** [reply]
I'm believe some of projects was closed or deleted so commits history was also deleted.
Or/And authors of this projects was removed from platform.

**Comment by igavelyuk** [comment]
I'm believe some of projects was closed or deleted so commits history(and pull history) was also deleted.
Or/And authors of this projects was removed from platform.
